extensions/data-transfer/portability-data-transfer-google/src/test/java/org/datatransferproject/datatransfer/google/media/GoogleMediaExporterTest.java

```diff
@@ -388,7 +390,8 @@ public void onlyExportAlbumlessPhoto()
   @Test
   public void testGetGoogleMediaItemSucceeds() throws IOException, InvalidTokenException, PermissionDeniedException {
     String mediaItemID = "media_id";
-    when(photosInterface.getMediaItem(any())).thenReturn(setUpSingleMediaItem(mediaItemID, mediaItemID, null));
+    MediaMetadata mediaMetadata = new MediaMetadata();
+    when(photosInterface.getMediaItem(any())).thenReturn(setUpSingleMediaItem(mediaItemID, mediaItemID, mediaMetadata));
```

Comment:
could we put this in your test? its better to define the "when" and stuff inside of the test, so we know exactly how the APIs are behaving

Reply:
Not sure since this is already inside the test. Did you highlight the wrong lines?

Reply:
yeah, its the media metadata and the "when(photosInterface...).." 
you can just leave the setup unchanged and move this to the top of your tests.

Reply:
oops i was so wrong so sorry